rewrite-gradle/src/test/java/org/openrewrite/gradle/RemoveEnableFeaturePreviewTest.java

```diff
@@ -0,0 +1,160 @@
+class RemoveEnableFeaturePreviewTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new RemoveEnableFeaturePreview("ONE_LOCKFILE_PER_PROJECT"));
+    }
+
+    @Test
+    void singleQuotes() {
+        //language=gradle
+        rewriteRun(
+          settingsGradle(
+            """
+                pluginManagement {
+                    repositories {
+                        gradlePluginPortal()
+                    }
+                }
+              
+                rootProject.name = 'merge-service'
+                enableFeaturePreview('ONE_LOCKFILE_PER_PROJECT')
+              """,
+            """
+                pluginManagement {
+                    repositories {
+                        gradlePluginPortal()
+                    }
+                }
+              
+                rootProject.name = 'merge-service'
+              """
+          )
+        );
+    }
+
+    @Test
+    void doubleQuotes() {
+        //language=gradle
+        rewriteRun(
+          settingsGradle(
+            """
+              pluginManagement {
+                  repositories {
+                      gradlePluginPortal()
+                  }
+              }
+              
+              rootProject.name = 'merge-service'
+              enableFeaturePreview("ONE_LOCKFILE_PER_PROJECT")
+              
+              include 'service'
+              """,
+            """
+              pluginManagement {
+                  repositories {
+                      gradlePluginPortal()
+                  }
+              }
+              
+              rootProject.name = 'merge-service'
+              
+              include 'service'
+              """
+          )
+        );
+    }
+
+    @Nested
+    class NoChange {
+        @Test
+        void differentFeature() {
+            //language=gradle
+            rewriteRun(
+              settingsGradle(
+                """
+                  pluginManagement {
+                      repositories {
+                          gradlePluginPortal()
+                      }
+                  }
+                  
+                  enableFeaturePreview("DIFFERENT_FEATURE")
+                  
+                  rootProject.name = 'merge-service'
+                  
+                  include 'service'
+                  """
+              )
+            );
+        }
+
+        @Test
+        void noArgument() {
+            //language=gradle
+            rewriteRun(
+              settingsGradle(
+                """
+                  pluginManagement {
+                      repositories {
+                          gradlePluginPortal()
+                      }
+                  }
+                  
+                  enableFeaturePreview()
+                  
+                  rootProject.name = 'merge-service'
+                  
+                  include 'service'
+                  """
+              )
+            );
+        }
+
```

Comment:
```suggestion
```

Personally, I would drop this test. If you were to attempt this script via Gradle, it would result in an error as this method doesn't exist.